ansible/plugins/callback/atmo_logger.py.j2

```diff
@@ -0,0 +1,93 @@
+DOCUMENTATION = '''
```

Comment:
At the top of this file you should mention that it is templated.

Reply:
I wonder if the template module automatically can do this with the configure ansible template message...

Reply:
This file is templated using the `./configure` script, not the ansible template module. I can change it to use the ansible template module and add `{{ ansible_managed }}` to the top of the file because that is a variable defined by default in `ansible.cfg` to show that a file is templated.

I used the configure script to keep it consistent with the `ansible.cfg`. Maybe we could make this change in a separate PR? It would require a few modifications to Clank, possibly a new role

Reply:
I made the change but the diff is still showing 🤔

Reply:
I see it